runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -547,6 +577,36 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 		}
 	}
 
+	@Subscribe
+	private void onGameTick(GameTick event)
+	{
+		long currentTime = System.currentTimeMillis();
+		boolean dailyReset = !loggingIn && currentTime - lastReset > ONE_DAY;
+
+		if ((dailyReset || loggingIn)
+			&& client.getVar(VarClientInt.MEMBERSHIP_STATUS) == 1)
```

Comment:
`Client#getVar()` is deprecated, use `Client#getVarcIntValue()` instead.
```suggestion
			&& client.getVarcIntValue(VarClientInt.MEMBERSHIP_STATUS) == 1)
```

Reply:
This doesn't look resolved to me, perhaps you incorrectly resolved this branch's merge conflict. Please be sure you have resolved a review comment before resolving the conversation in GitHub's UI.

Reply:
This should be correct now